src/hooks/useSidePanel.ts

```diff
@@ -64,7 +64,7 @@ function useSidePanel() {
     const {windowWidth} = useWindowDimensions();
     const sidePanelWidth = shouldUseNarrowLayout ? windowWidth : variables.sideBarWidth;
 
-    const [isSidePanelTransitionEnded, setIsSidePanelTransitionEnded] = useState(false);
+    const [isSidePanelTransitionEnded, setIsSidePanelTransitionEnded] = useState(true);
```

Comment:
I changed this to true because the side panel is not transitioning at all initially. This is to fix an issue where the edit composer won't be refocused after closing the emoji picker because ready to focus promise is never resolved.
https://github.com/Expensify/App/blob/11385b89542b20e14342c71eaf84aae57ba5d97f/src/libs/ComposerFocusManager.ts#L203-L211

The promise is created because the help modal is mounted even though it shouldn't be.
https://github.com/Expensify/App/blob/11385b89542b20e14342c71eaf84aae57ba5d97f/src/components/SidePanel/HelpModal/index.tsx#L45-L62
https://github.com/Expensify/App/blob/11385b89542b20e14342c71eaf84aae57ba5d97f/src/components/SidePanel/index.tsx#L5-L15

Reply:
But mWeb still doesn't refocus because of the early return here:
https://github.com/Expensify/App/blob/11385b89542b20e14342c71eaf84aae57ba5d97f/src/pages/home/report/ReportActionItemMessageEdit.tsx#L525-L531

Reply:
Is this an existing bug or a result of our change. Have we tested if it is causing any regression?

Reply:
It's actually a bug from https://github.com/Expensify/App/pull/61884. Before the early return added here, `setIsSidePanelTransitionEnded(true)` will be called after it's mounted and the animation is done (even though technically there is nothing to animate), so the value will eventually be true too.
https://github.com/Expensify/App/blob/d4347a6b3d05003355b08e109ac02fd0b0853a52/src/hooks/useSidePanel.ts#L76-L96

Reply:
We now have an [issue](https://github.com/Expensify/App/issues/62174) created for that bug.

Reply:
The offending PR is reverted, should I revert this back?

Reply:
Yes please.

Reply:
Can you check if syncing with main fixes it?

Reply:
Syncing with main and reverting this change fixes it.